app/src/main/res/layout/activity_album_review.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              xmlns:tools="http://schemas.android.com/tools"
+              android:orientation="vertical"
+              android:weightSum="1"
```

Comment:
im not sure, i think i used some of the code from my layout row to make this and that might have been in there, or it was in there when i generated the file, ill change it now